### PR TITLE
Run scene when receive message on MQTT

### PR DESCRIPTION
**Pull request by callemand**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to the community ([french forum](https://community.gladysassistant.com/)/[english forum](https://en-community.gladysassistant.com/)) for testing before merging.

### Description of change

Launch scene when receive MQTT message on channel "gladys/scene/[sceneSelector]" 
The message must contain "START'

### CODE REVIEW

File: server/lib/scene/index.js [hunk 1]
```diff
@@ -45,6 +47,7 @@ const SceneManager = function SceneManager(
   this.http = http;
   this.gateway = gateway;
   this.brain = brain;
+  this.service = service;
```
Comment (Pierre-Gilles) [inline server/lib/scene/index.js]: I'm not sure we want to inject services into the "scene" instance.

I think we should keep the same flow as the current MQTT integration:

- Only modify the MQTT integration
- Add a listener in the MQTT integration, and then emit an event (to start a scene)

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
As said on the forum, I think the topic should be: 

```
gladys/master/scene/[sceneSelector]/start
```

File: server/lib/scene/index.js (hunk 1 above)
Comment (callemand) [reply server/lib/scene/index.js]: You are right, I have modified my code to use `handleGladysMessage()`

**Review by Pierre-Gilles — APPROVED** [review]
Way simpler like this, and it works perfectly on my side 👌

**Pull request merged**